src/brevitas_examples/llm/main.py

```diff
@@ -174,6 +182,15 @@
         'sharded_torchmlir_group_weight',
         'sharded_packed_torchmlir_group_weight'],
     help='Model export.')
+parser.add_argument(
+    '--checkpoint-name',
+    type=str,
+    default=None,
+    help="Filename to save checkpoint. If `None`, no checkpoint is saved (default: %(default)s)")
+add_bool_arg(
+    parser, 'use-ocp', default=False, help='Use OCP format for float quantization. Default: False')
```

Comment:
Let's merge this and then I'll update the entrypoint here same style as the stable diffusion one in #971